packages/react-native/src/primitives/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Text, View } from 'react-native';
+
+import { icons } from '../../assets';
+import { Icon } from '../Icon';
+import { IconButton } from '../IconButton';
+
+import { ErrorMessageProps } from './types';
+import { styles } from './styles';
+
+export default function Label({
```

Comment:
Guessing this is meant to a have a different name 😏

Reply:
Oh wow, embarrassing copy/paste mistake 🙈